.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKDurationParser.cs

```diff
@@ -38,10 +38,10 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refDate)
                 }
 
                 var unitStr = unitResult.Unit;
-                var numStr = unitResult.Number;
+                var number = string.IsNullOrEmpty(unitResult.Number) ? 1 : double.Parse(unitResult.Number);
```

Comment:
On all number parsing, please specify the invariant culture.

Reply:
ok, done.